data/Audiences.ts

```diff
@@ -2,22 +2,22 @@ import { Audience } from "../types/targeting";
 
 const audiences :  {[id: string] : Audience } = {
   all_english: {
-    name: "All English users",
-    description: "All users in en-* locales.",
-    targeting: "localeLanguageCode == 'en'",
-    desktop_telemetry: "STARTS_WITH(environment.settings.locale, 'en')"
+    name: "All English users (release)",
+    description: "All users in en-* locales using the release channel.",
+    targeting: "localeLanguageCode == 'en' && browserSettings.update.channel == 'release'",
+    desktop_telemetry: "STARTS_WITH(environment.settings.locale, 'en') AND normalized_channel = 'release'"
   },
   us_only: {
     name: "US users (en)",
```

Comment:
Q: Should this explicitly mention “release” channel as well? (The other two do in the `name` as well as description)

Reply:
Yes, thanks!